modules/drivers/bigquery-cloud-sdk/src/metabase/driver/bigquery_cloud_sdk/params.clj

```diff
@@ -0,0 +1,96 @@
+(ns metabase.driver.bigquery-cloud-sdk.params
+  (:require [clojure.tools.logging :as log]
+            [java-time :as t]
+            [metabase.util.date-2 :as u.date])
+  (:import [com.google.cloud.bigquery QueryJobConfiguration$Builder QueryParameterValue StandardSQLTypeName]))
+
+#_(defn- param-type-old ^QueryParameterType [^String type-name]
```

Comment:
what's the purpose of these three commented out forms?

Reply:
Ugh, more dead code I forgot to remove.  Should be fixed in newest commit.